### ISSUE TITLE
Venom (and fen)

### ISSUE THREAD

**Issue by sange9402** [123Venom/plugin.video.venom#189]
Not syncing w trakt though I've reauthorized several times...

HTTPSConnectionpool(host='api.trakt.tv' , port=443)

Won't update as watched after watching something on fen like it used to either.

Please help. I reauthorized trakt...not sure what else to do.

**Comment by TapMeDead** [comment]
Try clearing cache, If that doesn't help, post your venom.log file here, just drag/drop it on the msg field.

**Comment by sange9402** [comment]
In which way do I clear cache? The only items I get on my log say "alarm clock"

**Comment by 123Venom** [comment]
where's the log?  Please follow the readme rules!!

**Comment by 123Venom** [comment]
you also provided none of the required details per rules either

**Comment by sange9402** [comment]
Not sure how to copy it lemme see if I can take a pic

**Comment by sange9402** [comment]
Venom- Logfile

redirection URI used in the authorization request, or was issued to another client')

[2022-04-09 15:38:35] kom Me: Re-Authenticating Trakt Token

[2022-04-09 15:38:37]

#:120

From func name: resources.lib.modules.trakt.re_auth() Line

msg : Please Re-Authorize your Trakt Account: 400 : ('error": "invalid_grant", 'error_description: The provided authorization grant is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.') [2022-04-09 15:38:37] From func name: resources.lib.modules.trakt.re_auth() Line #:120

nce with this

[2022-04-09 15:38:37] WIKom Re-Authenticating Trakt Token

msg : Please Re-Authorize your Trakt Account 400: ('error': 'invalid_grant, 'error_description: The provided authorization grant is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.") #:120

[2022-04-09 15:54:07] Mcomalvet Re-Authenticating Trakt Token

(2022-04-09 15:54:09) : From func name: resources.lib.modules.trakt.re_auth0 Line

msg: Please Re-Authorize your Trakt Account: 400: ('error: 'invalid_grant', 'error_description': 'The provided authorization grant is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.')

12022-04-09 15:54:101 Vinum INL: Re-Authenticating Trakt Token

**Comment by sange9402** [comment]
![Uploading PXL_20220411_025102636.MP.jpg…]()

**Comment by 123Venom** [comment]
open Venom settings.  Then on the Trakt category the first option is `Open My Accounts Addon`.  Once My Accounts settings comes up `Revoke` your trakt account.  Then re-auth.  Be sure to save via `OK` in My Accounts.  When Venom settings pops back up you MUST make sure to then hit `OK` there as well to save the sync data back to Venom's settings.

**Comment by sange9402** [comment]
Ok I tried this before. Let me try again. Thank you will let you know if it pops back up....

**Comment by 123Venom** [comment]
If this returns `HTTPSConnectionpool(host='api.trakt.tv' , port=443)` I'm afraid I'm at a loss to know why and will not be able to solve this for you.  I have 4 devices here and never get this error yet a few people seem to be getting it and I have no idea why.  If it returns I'd prob suggest trying another addon because I just don't know why it happens to a select few.  The last user that had connect issue we bumped the timeout value and it seemed to cure it for him.

**Comment by 123Venom** [comment]
also usually that `HTTPSConnectionPool(host='api.trakt.tv', port=443)` is accompanied by more info like...

`ReadTimeout: HTTPSConnectionPool(host='api.trakt.tv', port=443): Read timed out. (read timeout=20)` but it looks like a copy/paste butcher job done to the log line.  The normal method is to post the `venom.log` file and let ME read it!

You can check it out more yourself at this point but if it is a timeout error I can not cure that beyond what's already been done.
